docs/src/1_Mechanics/05_Vulnerabilities.yaml

```diff
@@ -9,7 +9,7 @@ Template: # Power name        # Must be unique
         Power:                # Must match unique power name
     Description: Descrip      # Flavor: 1-2 sentences             
     Tags: One or list
-    # StatOverride: AGL -2   # One of AGL, STR, Finesse, etc. followed by number
+    # StatAdjust: AGL -2   # One of AGL, STR, Finesse, etc. followed by number
```

Comment:
Does StatAdjust default to Bump vs. Override if not indicated? Or is it always 

StatAdjust:
    StatBump:
        AGL: -1